Filters/exceptions.txt

```diff
@@ -3,6 +3,8 @@
 ! `*-dnsotls-ds.metric.gstatic.com` (for instance, `a5a6380f-dnsotls-ds.metric.gstatic.com`).
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/81770
+@@||tag.partner.mairdumont.com^|
```

Comment:
Not sure if it is a good idea. Better try `$replace` rule.

Reply:
@Sergey-Lyapin try this `||kachelmannwetter.com/js/function.js$replace=/{adblocker='Enabled';/{adblocker='Disabled';/`
Also check another their domains.

Reply:
https://github.com/AdguardTeam/AdguardFilters/commit/ed1d141e472be0556b3b18d7ac6e87a7aa0af8c2 :man_facepalming: